### PR TITLE
Size t macos fix

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: malign_buffer.cc [hunk 1]
```diff
@@ -114,6 +118,13 @@ size_t MalignBuffer::RoundUpToPageSize(size_t k) {
   return ((k + kPageSize - 1) / kPageSize) * kPageSize;
 }
 
+size_t CacheLineSize() {
+    size_t lineSize = 0;
```
Comment (scal444) [inline malign_buffer.cc:123]: I think this needs to be in the apple #ifdef , might as well put it where it's used in that case

File: cpu_check.cc [hunk 2]
```diff
@@ -714,8 +715,8 @@ void Worker::Run() {
 
     LOG_EVERY_N_SECS(INFO, 30) << Jstat(
         Json("elapsed_s", static_cast<uint64_t>(TimeInSeconds() - t0)) + ", " +
-        Json("failures", errorCount.load()) + ", " +
-        Json("successes", successCount.load()) + ", " + Writer() +
+        Json("failures", static_cast<double>(errorCount.load())) + ", " +
```
Comment (scal444) [inline cpu_check.cc]: Why double for these two?

Comment (ghost) [reply cpu_check.cc]: I thought load instance returns a double type and because it doesn't compile.. Giving error: error: call to 'Json' is ambiguous

Comment (ghost) [reply cpu_check.cc]: i updated see below

File: malign_buffer.cc (hunk 1 above)
Comment (ghost) [reply malign_buffer.cc:123]: i updated see below

**Pull request closed without merging**